data/json/construction.json

```diff
@@ -931,7 +931,7 @@
     "qualities": [ [ { "id": "SMOOTH", "level": 1 } ] ],
     "components": [ [ [ "concrete", 2 ] ], [ [ "water", 2 ] ] ],
     "pre_terrain": "t_pit_shallow",
-    "post_terrain": "t_concrete"
+    "post_terrain": "t_thconc_floor"
```

Comment:
This floor comes with a roof.  Is that your intention?   Would some kind of mechanical device need to sit on top of the roof in this case?

Reply:
sorry for long inactivity but no that was not my intention
I really just wanted to make the concrete floor the player can make, the same that appears under a metal gate